.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,59 @@
+name: Lighthouse test
+
+on:
+  repository_dispatch:
+    types: [preview-created]
+
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+
+      - name: Install dependencies
+        run: npm install && npm install -g @lhci/cli@0.14.x
+
+      - name: Run Lighthouse
+        run: lhci autorun --collect.url=https://pr${{ github.event.client_payload.number }}.rs.school
```

Comment:
Can I use `npx lhci` here and remove `npm install -g @lhci/cli@0.14.x` from the previos step?

Reply:
I tried and there is no benefits, workflow execution time becomes even longer. Same with npm-install action. 
And the method used here is recommended from lighthouse docs.